components/cards/card.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
```

Comment:
One more thing to clarify about: This branch is related to input & image right then the card related materials are also showing up so you can edit the name of the Pull request as cards,Input&image